x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/index.scss

```diff
@@ -3,3 +3,9 @@
 .fleet__epm__shiftNavTabs {
   margin-left: $euiSize * 6 + $euiSizeXL * 2 + $euiSizeL;
 }
+
+@media (max-width: 767px) {
+  .header_container .euiFlexItem {
+    margin-bottom: 0 !important;
+  }
+}
```

Comment:
I’m not up to speed on the latest recommendations from the EUI / design team. I know they’re generally trying to move towards CSS in JS, should we be using `styled-components` here instead of a separate CSS file? I believe we wouldn't need the `!important` with this option as well.